Assets/HoloToolkit/Utilities/Shaders/WindowOcclusion.shader

```diff
@@ -0,0 +1,51 @@
+///
+/// Simple occlusion shader that can be used to hide other objects.
+/// This prevents other objects from being rendered by drawing invisible 'opaque' pixels to the depth buffer.
+///
+Shader "HoloToolkit/WindowOcclusion"
+{
+    Properties
```

Comment:
Not sure if that'd work, but it's what Occlusion.shader does, which is already in HoloToolkit-Unity.